project-reports/2024/2024-Q3-Hyperledger-Identus.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: default
+title: 2024 Q3 Hyperledger Identus
+parent: 2024
+grand_parent: Project Updates
+nav_exclude: true
+---
+
+Created by Fayyaadh Adams with input from other maintainers
+
+
+# Project Health
+
+The project is fairly new to the Hyperledger Ecosystem however we continue to have steady contributors. We are currently not on the LFX dashboard yet and are waiting to be added to gain further insights. In the last three weeks alone, we have seen 3 new contributors (not code related) show up and additional usage of our code within production environments (https://socious.io/ - production use case). 
+
+
+# Questions/Issues for the TOC
+
+Is there a way to access funds for open source contributions and development? 
+
+
+# Releases
+
+  * Identus V2.12
+    * Identus Cloud Agent - V.1.33.0
+    * Identus Mediator - V.0.14.2
+    * Identus Apollo - V.1.2.14
+    * Identus Edge Agent SDK TS - V.5.1.0
+    * Identus Edge Agent SDK KMP - V.3.0.0
+    * Identus Edge Agent SDK SWIFT - V.6.0.0
+    * Identus Docs - V.1.92.0
+
+
+
+# Overall Activity in the Past Quarter
+
+The main focus this quarter, and continuing into the next, is to achieve a rich set of features within Identus. The contribution of the code base from IOG to Hyperledger was completed in early July 2024. Initially it was only the cloud agent respository that was part of Hyperledger that started out as the Open Enterprise Agent Lab. The rest of the repositories were public repositories in IOG and has since been migrated to Hyperledger. 
```

Comment:
```suggestion
The main focus this quarter, and continuing into the next, is to achieve a rich set of features within Identus. The contribution of the code base from IOG to Hyperledger was completed in early July 2024. Initially it was only the cloud agent repository that was part of Hyperledger that started out as the Open Enterprise Agent Lab. The rest of the repositories were public repositories in IOG and has since been migrated to Hyperledger. 
```